javascript/readme.md

```diff
@@ -2,20 +2,46 @@
 
 If you are new to JavaScript, you may want to visit our [Go Direct FAQ page](./godirect-js-faqs.md) for some tips and troubleshooting ideas.
 
-## Obtain godirect-js
-
-Install the [godirect-js npm package](https://www.npmjs.com/package/@vernier/godirect)
-
-## Get godirect-examples
-
-- [Downloading the repository as a zip file](https://github.com/VernierST/godirect-examples/archive/master.zip)
-- [Using Git to clone the repository](https://github.com/VernierST/godirect-examples.git)
-
-## godirect-sensor-readout
-
-The go-direct-sensor-readout example is a simple example to get started with. It demonstrates how to get default enabled sensor(s) and log value changes.
-
-Click [here](./godirect-sensor-readout/) to run it in your browser, or [here](./godirect-sensor-readout/index.html) to view the source code.
+The examples in this folder will walk you through some of the basics of talking to a Go Direct device using JavaScript. 
+
+For other ideas and more advanced examples, take a look in the [/advanced_examples/](./advanced_examples) folder.
+
+## Example 1: Showing Sensor Information
+The [gdx_getting_started_1.html](./gdx_getting_started_1.html) example shows how you can use the gdx functions to list information on all of the sensor channels on a Go Direct device. This includes sensor channel number, sensor description, units, and "incompatible sensors".
```

Comment:
No incompatible sensors in this one. At least, not yet

Reply:
Done